plugins/inputs/nfsclient/README.md

```diff
@@ -0,0 +1,39 @@
+# Telegraf plugin: NFSClient
+
+#### Plugin arguments:
+- **fullstat** bool: Collect per-operation type metrics
+
+#### Description
+
+The NFSClient plugin collects data from /proc/self/mountstats, by default it will only include a quite limited set of IO metrics.
+If fullstat is set, it will collect a lot of per-operation statistics.
+
+#### Measurements & Fields
+
+- nfsstat_read
```

Comment:
Can you format these like in the [example readme](https://github.com/influxdata/telegraf/blob/master/plugins/inputs/EXAMPLE_README.md#metrics)

Reply:
I think this is done...see comment below

Reply:
@hawson I agree, please mark as solved.